src/main/java/core/basesyntax/CsvFileReader.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvFileReader implements FileReaderService {
+    @Override
+    public List<String> readData(String filePath) throws IOException {
+        List<String> data = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(filePath))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                data.add(line);
+            }
+        }
+        return data;
+    }
```

Comment:
1. I can't see any logic specific for csv. Please remove csv from naming. FileReaderImpl would be fine
2. Please add catch block and throw RuntimeException there as shown in comment for FileWriter